Compile error on Windows due to unresolved winapi

# Environment
- Windows 10 21H2 (OS Build 19044.1466)
- target stable-x86_64-pc-windows-msvc
- rustc 1.58.1 (db9d1b20b 2022-01-20)
- cargo 1.58.0 (f01b232bc 2022-01-19)

# Procedure for reproducing

Create new cargo package.

``` sh
cargo new --edition 2018 hoge
```

Edit `Cargo.toml`.

``` toml
[package]
name = "hoge"
version = "0.1.0"
edition = "2018"

[dependencies]
keyring = "1"
```

Edit `src/main.rs`.

``` rs
extern crate keyring;

fn main() {
    println!("Hello, world!");
}
```

Build package and it produce following message.

``` sh
cargo build
```

```
   Compiling winapi v0.3.9
   Compiling byteorder v1.4.3
   Compiling keyring v1.0.0
error[E0432]: unresolved import `winapi::um::errhandlingapi`
  --> C:\Users\***\.cargo\registry\src\github.com-1ecc6299db9ec823\keyring-1.0.0\src\windows.rs:11:17
   |
11 | use winapi::um::errhandlingapi::GetLastError;
   |                 ^^^^^^^^^^^^^^ could not find `errhandlingapi` in `um`

For more information about this error, try `rustc --explain E0432`.
error: could not compile `keyring` due to previous error
```

Change edition to 2021, it produce following message.

```
    Updating crates.io index
   Compiling winapi v0.3.9
   Compiling keyring v1.0.0
error[E0432]: unresolved import `winapi::shared::winerror`
 --> C:\Users\***\.cargo\registry\src\github.com-1ecc6299db9ec823\keyring-1.0.0\src\windows.rs:7:21
  |
7 | use winapi::shared::winerror::{
  |                     ^^^^^^^^ could not find `winerror` in `shared`

error[E0432]: unresolved import `winapi::um::errhandlingapi`
  --> C:\Users\***\.cargo\registry\src\github.com-1ecc6299db9ec823\keyring-1.0.0\src\windows.rs:11:17
   |
11 | use winapi::um::errhandlingapi::GetLastError;
   |                 ^^^^^^^^^^^^^^ could not find `errhandlingapi` in `um`

For more information about this error, try `rustc --explain E0432`.
error: could not compile `keyring` due to 2 previous errors
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -18,7 +18,7 @@ secret-service = "2.0.1"
 
 [target.'cfg(target_os = "windows")'.dependencies]
 byteorder = "1.2.1"
-winapi = { version =  "0.3", features = ["wincred", "minwindef"] }
+winapi = { version =  "0.3", features = ["wincred", "winerror", "errhandlingapi", "minwindef"] }
 
 [dev-dependencies]
 rpassword = "5.0"

